include/mbgl/shaders/symbol_layer_ubo.hpp

```diff
@@ -46,20 +46,12 @@ struct alignas(16) SymbolDrawableUBO {
     /* 220 */ float pitch;
     /* 224 */ /*bool*/ int rotate_symbol;
     /* 228 */ float aspect_ratio;
-    /* 232 */ std::array<float, 2> pad;
+    /* 232 */ float fade_change;
+    /* 236 */ float pad;
     /* 240 */
 };
 static_assert(sizeof(SymbolDrawableUBO) == 15 * 16);
 
-/// Dynamic UBO
-struct alignas(16) SymbolDynamicUBO {
```

Comment:
This was broken out into its own UBO for performance, IIRC, is it no longer better/relevant?

Reply:
SymbolDynamicUBO is not implemented in the shader code, that's why the symbol render tests were failing.

Reply:
My fix was to add the `fade_change` from `SymbolDynamicUBO` to `SymbolDrawableUBO`. But I can implement `SymbolDynamicUBO` in the shader code instead.

Reply:
What is the performance improvement here? Both UBOs are updated in the same place.

Reply:
I believe that value changes every frame, and the benefit was that it could be updated once and shared with all the drawables rather than updated separately for each drawable's UBO.

I guess that benefit would obviated (at least on Metal) by having the UBO bind as bytes rather than an explicit buffer.

Reply:
I see that was implemented in GL, so I will implement in Metal too. Still not clear what is the improvement now, not in the future, the split in this moment seems to decrease performance from my point of view, because is updating the same as a single UBO and we have 2, instead of 1, also the size is bigger, `fade_change` could occupy the pad space.

Reply:
My point is maybe we should split this UBO when we do also the improvement too, like update less often.

Reply:
Reverted.